docs/content/en/docs/reference/artifacts.md

```diff
@@ -543,12 +558,12 @@ These steps use `image-builder` to create a Ubuntu-based image for Nutanix AHV a
 
       * `--os`: `ubuntu`
       * `--hypervisor`: For Nutanix use `nutanix`
-      * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23, and 1-24.
+      * `--release-channel`: Supported EKS Distro releases include 1-21, 1-22, 1-23, 1-24 and 1-25.
```

Comment:
Same question about 1-21.

Reply:
Leaving this until deprecation in next minor release.